HTTPServerProject/HTTPServerResponse/ResponseByPath.cs

```diff
@@ -7,8 +7,8 @@
 using HTTPServerProject.Responses.Write;
 using HTTPServerProject.WriteStream;
 
-namespace HTTPServerProject.Path
-{
+namespace HTTPServerProject.Path;
+
```

Comment:
It's not part of this PR - but I noticed it looking at this one. RequestPath doesn't need a default and probably shouldn't use one. You're always injecting a dictionary - which is a good thing! - so you probably don't want the user to be able to create a RequestPath without a dictionary. It would be empty, and essentially useless.

Reply:
Agreed.